Makefile

```diff
@@ -17,7 +17,7 @@ browser-tests: on_host
 	pushd tests/ui ; ./node_modules/.bin/intern-runner config=intern-local d=developer.allizom.org b=firefox; popd
 
 clean:
-	find kuma -name '*.pyc' -exec rm {} \;
+	find . -name '*.pyc' -exec rm {} \;
```

Comment:
You'll need to exclude or restore ``vendor/src/python-magic/magic.pyc``, which is (probably incorrectly) checked into the the python-magic repo.